optimade/server/routers/utils.py

```diff
@@ -26,6 +28,35 @@
 }
 
 
+def optional_base_urls(
```

Comment:
this can now be deleted, right?

Reply:
He, yeah, that was supposed to have been removed in that commit - it now is :)